R/system.R

```diff
@@ -1482,3 +1482,15 @@ read_rds_file <- function(file, refhook = NULL){
     readRDS(file, refhook)
   }
 }
+
+#'Wrapper for readr::read_lines to support vector to data frame conversion
+#'@export
+read_raw_lines <- function(file, locale = readr::default_locale(), na = character(),
+                            skip = 0, n_max = Inf, progress = interactive()){
+  loadNamespace("readr")
+    # if it's local file simply call readr::read_delim
```

Comment:
Is this comment relevant? I don't see read_delim around here.

Reply:
removed it.